unstructured/ingest/connector/chroma.py

```diff
@@ -29,8 +29,8 @@ class SimpleChromaConfig(BaseConnectorConfig):
     access_config: ChromaAccessConfig
     collection_name: str
     path: t.Optional[str] = None
-    tenant: t.Optional[str] = None
-    database: t.Optional[str] = None
+    tenant: t.Optional[str] = "default_tenant"
+    database: t.Optional[str] = "default_database"
```

Comment:
what's going on here? why do we have these defaults??

Reply:
I mean, and likewise the defaults [here](https://github.com/Unstructured-IO/unstructured/blob/f1b8ceff30a16e21d93403b1504571fea092583a/unstructured/ingest/cli/cmds/chroma.py#L27-L41)?

Reply:
seems like these fields should just be required, right?